sc/social/like/tests/test_upgrades.py

```diff
@@ -189,7 +190,7 @@ def test_register_cover_tiles(self):
         self.execute_upgrade_step(step)
 
         registered = api.portal.get_registry_record('plone.app.tiles')
-        [self.assertIn(t, registered) for t in TILES]
+        [self.assertIn(t, registered) for t in TILES]  # noqa: E501; pylint: disable=W1662
```

Comment:
Here, the same reasoning as  sc/social/like/plugins/facebook/utils.py.

Reply:
the fix has not being released yet: https://pypi.org/project/pylint/

Reply:
You're right. Although unrelated to this PR, that's why we still have these comments in the IDG project as well. I thought they created a release since you said in a comment it's working in the related issue.

Reply:
I'm going to ping them there.